### ISSUE TITLE
The `indentation` rule never measures the line a value opens on behind a break standing in the declaration's `raws.between`

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#635]
### What minimal example or steps are needed to reproduce the bug?

Where a declaration's value opens on the line behind its colon, the line break stands in the declaration's `raws.between` rather than in its value, and `indentation` measures the value's line by nothing: `checkValue` asks whether the value holds a break before it reads the declaration, and the value holds none.

```css
a {
	b:
1px;
}
```

Under `"tab"` that draws nothing, at whatever depth the line stands, where the same value broken behind its first word is reported:

| input | syntax | warnings |
| --- | --- | --- |
| `a {⏎\tb:⏎1px;⏎}` | css, scss, less | none |
| `a {⏎\tb:⏎\t\t\t1px;⏎}` | css | none |
| `a {⏎\tb: /* c */⏎1px;⏎}` | css | none |
| `a {⏎\tb: c⏎1px;⏎}` | css | `3:1 Expected indentation of 2 tabs` |
| `<a style="b:⏎1px"></a>` | html | none |

Measured on `3fa4b43`, the branch of #594, and the same on `63d255f`, the commit it stands on.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/indentation": "tab"
	}
}
```

### What did you expect to happen?

The line the value opens on is measured like any other line of the value, a level deeper than the declaration, wherever the parser filed the break in front of it.

### What actually happened?

The line is measured by nobody. The text `declarationString` hands over holds the break, and the writer already puts a position falling inside `raws.between` into that raw; what stands in front of both is the guard on the value alone.

### Notes

Found while measuring #594 over `style` attributes, where every declaration is written on the tag's line and a value pushed onto the next one is the shape that shows it. The guard is the one Stylelint's own rule carried.
